resources/fixtures/Litecraft/Litecraft-WashX.36.qxf

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE FixtureDefinition>
+<FixtureDefinition xmlns="http://www.qlcplus.org/FixtureDefinition">
+ <Creator>
+  <Name>Q Light Controller Plus</Name>
+  <Version>4.12.1</Version>
+  <Author>Henrik Bauer, Jannis Achstetter</Author>
+ </Creator>
+ <Manufacturer>Litecraft</Manufacturer>
+ <Model>WashX.36</Model>
+ <Type>Color Changer</Type>
+ <Channel Name="Strobe functions">
+  <Group Byte="0">Shutter</Group>
+  <Capability Min="0" Max="19" Preset="ShutterClose">Shutter closed</Capability>
+  <Capability Min="20" Max="24" Preset="ShutterOpen">Shutter open</Capability>
+  <Capability Min="25" Max="64" Preset="StrobeSlowToFast">Strobe slow -&gt; fast</Capability>
+  <Capability Min="65" Max="69" Preset="ShutterOpen">Shutter open</Capability>
+  <Capability Min="70" Max="84" Preset="RampUpSlowToFast">Ramp up</Capability>
+  <Capability Min="85" Max="89" Preset="ShutterOpen">Shutter open</Capability>
+  <Capability Min="90" Max="104" Preset="RampDownSlowToFast">Ramp down</Capability>
+  <Capability Min="105" Max="109" Preset="ShutterOpen">Shutter open</Capability>
+  <Capability Min="110" Max="124" Preset="StrobeRandomSlowToFast">Strobe random</Capability>
+  <Capability Min="170" Max="184">Strobe exploding</Capability>
+ </Channel>
+ <Channel Name="Dimmer" Preset="IntensityMasterDimmerFine"/>
+ <Channel Name="Control">
+  <Group Byte="0">Maintenance</Group>
+ </Channel>
+ <Channel Name="Red" Preset="IntensityRed"/>
+ <Channel Name="Green" Preset="IntensityGreen"/>
+ <Channel Name="Blue" Preset="IntensityBlue"/>
+ <Channel Name="White" Preset="IntensityWhite"/>
+ <Mode Name="7 channel">
+  <Channel Number="0">Strobe functions</Channel>
+  <Channel Number="1">Dimmer</Channel>
+  <Channel Number="2">Control</Channel>
+  <Channel Number="3">Red</Channel>
+  <Channel Number="4">Green</Channel>
+  <Channel Number="5">Blue</Channel>
+  <Channel Number="6">White</Channel>
+ </Mode>
+ <Physical>
+  <Bulb Type="" Lumens="0" ColourTemperature="0"/>
+  <Dimensions Weight="0" Width="0" Height="0" Depth="0"/>
```

Comment:
Invalid. Please fix.

Reply:
True, I didn't have the manual at that time (impossible to find online ...). Fixture is completed now